### PR TITLE
account_auto_fy_sequence : When create journal, use fy instead of year in the sequence

### PR DESCRIPTION
**Pull request by laetitia-gangloff**
Hello,

I rewrite the method 'create_sequence' from account.journal to be able to set an other prefix.

### CODE REVIEW

File: account_auto_fy_sequence/models/account_journal.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+##############################################################################
+
+from openerp.osv import orm
+
+
+class account_journal(orm.Model):
+    _inherit = "account.journal"
+
+    def create_sequence(self, cr, uid, vals, context=None):
+        """ Create new no_gap entry sequence for every new Joural
```
Comment (sbidoul) [inline account_auto_fy_sequence/models/account_journal.py:33]: Thanks @laetitia-gangloff. Would it not be more robust to first call super.create_sequence then replacing %(year)s in the created sequence?

**Comment by sbidoul** [comment]
Can you also add a line in the README? 

"The module also replaces %(year)s by %(fy)s in the default prefix for new journals, assuming it is a safer default."

**Comment by laetitia-gangloff** [comment]
@sbidoul done, thank you !

**Comment by pedrobaeza** [comment]
I agree with @sbidoul that it's safer to call super.

**Comment by pedrobaeza** [comment]
And then browse + replace.

**Comment by laetitia-gangloff** [comment]
Right, I corrected in this way.

**Comment by pedrobaeza** [comment]
Thanks. :+1: when Travis passes.

**Comment by sbidoul** [comment]
Thanks @laetitia-gangloff. :+1:

**Comment by tafaRU** [comment]
code review and test LGTM, so :+1:  and I proceed with the merge.

**Pull request merged**